tests/tracer/test_propagation.py

```diff
@@ -853,7 +853,18 @@ def test_extract_tracecontext(headers, expected_context):
     # B3 headers
     (
         "valid_b3_simple",
-        [PROPAGATION_STYLE_B3],
+        [PROPAGATION_STYLE_B3_MULTI],
+        B3_HEADERS_VALID,
+        {
+            "trace_id": 5208512171318403364,
+            "span_id": 11744061942159299346,
+            "sampling_priority": 1,
+            "dd_origin": None,
+        },
+    ),
+    (
+        "test_deprecated_b3_style_still_works",
```

Comment:
I can't tell why this new test is failing. With the added deprecation in config.py we should be switching the value to `b3multi` if we see `b3` but I must be missing something.